index.html

```diff
@@ -17,41 +17,30 @@
     })(window,document,'script','dataLayer','GTM-5N6WR7G');</script>
 
     <meta charset="utf-8" />
-    <link rel="shortcut icon" href="%PUBLIC_URL%/make-sense-ico.png" />
+
+    <link rel="shortcut icon" href="/make-sense-ico.png" />
     <link href="https://fonts.googleapis.com/css?family=Saira+Semi+Condensed&display=swap" rel="stylesheet">
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <meta name="theme-color" content="#000000" />
+    
+    <meta name="theme-color" content="#171717" />
     <!--
       manifest.json provides metadata used when your web app is installed on a
       user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
     -->
-    <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
-    <!--
-      Notice the use of %PUBLIC_URL% in the tags above.
-      It will be replaced with the URL of the `public` folder during the build.
-      Only files inside the `public` folder can be referenced from the HTML.
-
-      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
-      work correctly both with client-side routing and a non-root public URL.
-      Learn how to configure a non-root public URL by running `npm run build`.
-    -->
+    <link rel="manifest" href="/manifest.json" />
     <title>Make Sense</title>
   </head>
   <body>
+    <style>
```

Comment:
It is actually pretty good idea to configure that color, but lets do it in css.

Reply:
yes, for that reason, I often add style inside body to reduce blank page when refreshing, so the `<style>` tag is actually intended

Reply:
So, you placed it here to load the background color right away and not wait for the css to be loaded?

Reply:
exactly. just as the app have dark background . whenever we reload or having bad connection. it will cause a long white flicker.